tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
This is not needed; it was added in an attempt to fix a problem when importing `index.ts` that was resolved via the previous PR instead.

```suggestion
```